test/unit/detail/format_help_test.cpp

```diff
@@ -536,35 +536,3 @@ TEST_F(format_help_test, copyright)
              + license_text;
     EXPECT_EQ(get_parse_cout_on_exit(parser), expected);
 }
-
-TEST_F(format_help_test, subcommand_parser)
-{
-    int option_value{};
-
-    auto parser = get_subcommand_parser({"-h"}, {"sub1", "sub2"});
-    parser.info.description.push_back("description");
-    parser.add_option(option_value, sharg::config{.short_id = 'f', .long_id = "foo", .description = "foo bar."});
```

Comment:
sorry, whats wrong here now? I remember you said it must be deleted but I forget and can't see why

Reply:
Originally because options are no longer allowed.
However, I now changed it to just use a `flag`; and also changed `The following options below` to `The following options`